ccl_chromium_indexeddb.py

```diff
@@ -944,6 +950,7 @@ class WrappedIndexDB:
     is simpler and more pythonic.
     """
     def __init__(self, leveldb_dir: os.PathLike, leveldb_blob_dir: os.PathLike = None):
+        leveldb_blob_dir = leveldb_blob_dir or leveldb_dir.with_suffix(".blob")
```

Comment:
I favour specificity of the caller passing the blob path over this default behaviour I think.

Reply:
I respectfully disagree. This is easy to derive data that WrappedIndexDB can easily figure out. Why would we burden callers with doing this scut work, when it is easy to derive like this as a default parameter?

We could eliminate the parameter entirely, removing likely never needed flexibility?

Reply:
I'll counter this by saying: I used this flexibility last week when I had a blob folder containing corrupt data recovered from a disk image. There might also be situations where the blob folder straight up doesn't exist and as a caller I'd want to know about it early rather than deep within another call. Explicit is better than implicit, etc. 

It's I think it's worth bearing in in that the original reason for this module to exist is for digital forensics and research where this flexibility is sometimes desirable and where precision and traceability is always going to trump the minor inconvenience of checking for a blob folder's presence and adding it in my eyes.